charts/hivemq-operator/values.yaml

```diff
@@ -77,21 +77,23 @@ monitoring:
 hivemq:
   # Custom container spec to add to the HiveMQ Pod
   sidecars: []
-  # Custom init container specs to add to the HiveMQ Pod. This is an extension of the initialization field. In comparison this field does not have any defaults but allows for more granular configuration using the full K8s Container API
+  # Custom init container specs to add to the HiveMQ Pod. This is an extension of the initialization field. In comparison, this field does not have any defaults but allows for more granular configuration using the full K8s Container API
   initContainers: []
   # HiveMQ Pod security context
+  # To configure see example https://github.com/hivemq/helm-charts/tree/master/container/README.md
```

Comment:
I would leave this example line out of the values.yaml. Not needed in this context.